packages/elements-react/src/theme/default/components/card/footer.tsx

```diff
@@ -46,24 +46,25 @@ function LoginCardFooter() {
 
   return (
     <>
-      {formState.current === "provide_identifier" && (
-        <span className="font-normal leading-normal antialiased text-interface-foreground-default-primary">
-          {intl.formatMessage({
-            id: "login.registration-label",
-            defaultMessage: "No account?",
-          })}{" "}
-          <a
-            className="text-button-link-brand-brand transition-colors hover:text-button-link-brand-brand-hover underline"
-            href={initFlowUrl(config.sdk.url, "registration", flow)}
-            data-testid={"ory/screen/registration/action/login"}
-          >
+      {formState.current === "provide_identifier" &&
+        config.project.registration_enabled && (
```

Comment:
@pcaillaudm previously, the registration link was hidden when registration was disabled. Your refactoring of the component missed that change, introducing regression https://github.com/ory/network/issues/281#issuecomment-2740028352

Could you please add a test either to elements directly, or an e2e test, to ensure that registration_disabled means there is no sign up link, to prevent future regressions? Thank you!